### DISCUSSION TITLE
Items don't fall through destroyed walls.

### DISCUSSION THREAD

**Discussion by tribalknight** [FakeFishGames/Barotrauma#14054] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Upon spawning an item through the debug console above a completely destroyed wall, the item itself does not pass through. Experimented by spawning several metal crates onto a damaged wall, which resulted in the metal crates staying on the wall. Important to note, any items that are thrown can pass through walls as normal. Items like nitroglycerin can pass through the walls when thrown, but do not appear to go through the walls when spawned/dropped onto them.

### Reproduction steps

1. Enter sub-editor or create a multiplayer server and go onto any ship.
2. Start the round/ship and destroy a part of a horizontal wall.
3. Attempt to place items onto the destroyed part of the wall (away from any solid parts of it),
4. Any items that are dropped or spawned over the destroyed wall will act like the wall still exists and collide with the wall.
![20240529155729_1](https://github.com/FakeFishGames/Barotrauma/assets/34112533/d9859b00-dd04-49fb-a780-a1b95399f876)

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Tbh, I'm not sure they should actually drop. I think it could be frustrating if all the items at the bottom of the sub dropped out of the sub when there's a large breach.

**Reply by UnhappyWithMyUsername** [reply]
They should. Don't casualize the entire game. Items fell through floors all these years and some mods (like my enhanced reactors) made use of that. Now it's not nearly as funny or risky to drop an active fuel rod as it used to.

Funny incidents like these are no more...
https://www.youtube.com/shorts/xnF3oawkJZY

At least make it configurable in the item's body please.

**Reply by Regalis11** [reply]
Hmm, I didn't remember they used to fall through! 😅 In that case this is most likely not intentional, and definitely seems worth looking into.

**Comment by A2ur3** [comment]
need a fix

https://github.com/user-attachments/assets/fe1092fa-2b86-45f2-932f-416a88f02f81

**Reply by SoobGorl** [reply]
Ticketed this issue in particular at [#16910](https://github.com/FakeFishGames/Barotrauma/issues/16910)
